src/frontend/boilerplates/archipelago/src/resources/Agent/Actor/Organization/OrganizationShow.js

```diff
@@ -44,31 +45,27 @@ const OrganizationShow = props => (
       </Grid>
       <Grid item xs={12} sm={3}>
         <SideList>
-          <ReifiedArrayField
+          <GroupedArrayField
             source="pair:organizationOfMembership"
             groupReference="MembershipRole"
-            groupLabel="pair:label"
+            groupComponent={({record, ...otherProps })=>
+              <RightLabel label={record['pair:label']}/>
+            }
```

Comment:
Si on veut rester fidèle aux bonnes pratiques React-Admin, il faudrait passer juste `record`, pas un objet (d'ailleurs tu n'utilises pas otherProps)

Reply:
ok, pourtant pour les layout, c'est les props complètes qui sont passés. c'est record pour les field? Je fais la modif

Reply:
Oui. Voir par exemple SimpleList, parmi beaucoup d'autres exemples
https://marmelab.com/react-admin/List.html#the-simplelist-component